src/GridFieldHasOneEditButton.php

```diff
@@ -35,9 +35,9 @@ public function getHTMLFragments($gridField)
             $objectName = $singleton->i18n_singular_name();
 
             if ($record->exists()) {
-                $buttonName = _t(GridField::class . '.Edit', 'Edit {name}', ['name' => $objectName]);
+                $buttonName = _t('SilverStripe\Forms\GridField\GridField.Edit', 'Edit {name}', ['name' => $objectName]);
```

Comment:
By the way, I'd recommend that you don't re-use core translations, but define your own. That way if they change in core your strings here won't be arbitrarily updated.